UrlRedirects/UrlRedirects.csproj

```diff
@@ -11,8 +11,8 @@
     <ProjectTypeGuids>{349c5851-65df-11da-9384-00065b846f21};{fae04ec0-301f-11d3-bf4b-00c04f79efbc}</ProjectTypeGuids>
     <OutputType>Library</OutputType>
     <AppDesignerFolder>Properties</AppDesignerFolder>
-    <RootNamespace>UrlRedirects</RootNamespace>
-    <AssemblyName>UrlRedirects</AssemblyName>
+    <RootNamespace>Forte.UrlRedirects</RootNamespace>
```

Comment:
Should it maybe be EpiserverRedirects? There is nothing in the name of the package that indicates this is for Episerver.